pycbc/inference/io/base_hdf.py

```diff
@@ -181,7 +181,8 @@ def read_samples(self, parameters, array_class=None, **kwargs):
         # convert to FieldArray
         samples = array_class.from_kwargs(**samples)
         # add the static params
-        for (p, val) in self.static_params.items():
+        for (p, val) in self.static_params.items() + \
+                        self[self.samples_group].attrs.items():
```

Comment:
(PEP8) This is over indented (see the code climate report). I think reducing the indent to two tabs past the `for` might do it. Alternatively, create a variable above, say, `addattrs = ...`, then do `for (p, val) in addattrs:`.

Reply:
I did see the overindent complaint, but isn't this just a matter of style? I like the indentation the way I did it, rather than the way CodeClimate wants me to do it!  I will go for the option of creating a variable above.

Reply:
It still complains about the indentation, is that an issue that can be ignored?

Reply:
It's because when doing new line continuation, you should just use a single tab indent from the previous line:
```
addattrs = self.static_params.items() + \
    self[self.samples_group].attrs.items()
```

Alternatively,  if you want the second line to line up, stick in a parenthesis. Then you can also get rid of the line continuation character:
```
addattrs = (self.static_params.items() +
            self[self.samples_group].attrs.items())
```

This isn't just a style set by Code Climate, it's the Python standard style guide ([PEP-8](https://www.python.org/dev/peps/pep-0008/)). I'd like to stick to that (and update old code as we move along), so please change it to one of those two.

Reply:
Ok, I chose the parenthesis option and CodeClimate seems to like it now